config/initializers/faraday.rb

```diff
@@ -34,6 +34,7 @@
     status: env.status,
     service: service,
     name: 'request_metric.faraday',
+    log_filename: 'rails.log',
```

Comment:
is this the one that goes to `#{Rails.env}.log`? aka `production.log` usually?

Reply:
Yep. The important part is having some way to distinguish the log streams from one another, so it doesn't super matter what the name is for that aspect. We could name it whatever we like, definitely open to suggestions.

Reply:
I'd have it match the file name if possible!
```suggestion
    log_filename: "#{Rails.env}.log",
```

Reply:
I think that should work, let me double check.

Reply:
Since it's going to be the default, I've just removed it.